modules/support_ticket/src/Controller/SupportTicketController.php

```diff
@@ -193,7 +193,7 @@ public function revisionOverview(SupportTicketInterface $support_ticket) {
           '#context' => [
             'date' => $link,
             'username' => $this->renderer->renderPlain($username),
-            'message' => SafeMarkup::xssFilter($revision->revision_log->value),
+            'message' => ['#markup' => $revision->revision_log->value, '#allowed_tags' => Xss::getHtmlTagList()],
```

Comment:
Don't need to pass the allowed tags if they're not different. This would also allow dropping the use statement.